File: src/question/month07/day0707/ms_0707.cs
using System;

namespace algorithm_07
{
    class ms_0707
    {
        static void Main(string[] args)
        {
            string[] animal = { "dog", "cat", "chick" };

            Console.Write("Number? ");
            var num = Convert.ToInt32(Console.ReadLine());

            if(1 <= num && num <= 3)
                Console.WriteLine(animal[num-1]);
            else
                Console.WriteLine("I don't know");
        }
    }
}

File: src/question/month07/day0707/ms_0707.cpp
#include <iostream>

using namespace std;

enum ANIMAL { DOG = 1, CAT, CHICK };

int main() {
	int num;
	
	cout << "Number? ";
	cin >> num;

	switch (num) {
	case DOG:
		cout << "dog" << endl;
		break;
	case CAT:
		cout << "cat" << endl;
		break;
	case CHICK:
		cout << "chick" << endl;
		break;
	default:
		cout << "I don't know." << endl;
		break;
	}

	return 0;
}